src/Sulu/Bundle/MediaBundle/Tests/Unit/Media/ImageConverter/Scaler/ScalerTest.php

```diff
@@ -99,4 +100,13 @@ public function testScaleWithFloatWidth()
         $this->assertEquals(220, $image->getSize()->getWidth());
         $this->assertEquals(442, $image->getSize()->getHeight());
     }
+
+    private function createImagine()
```

Comment:
Have you introduced that function now, in order to test against both adapters? Or was there another reason for that?

Reply:
The first reasons was that gd was not available and couldn't be installed (not sure why).
So I installed Imagick at the end I think its good that we test agains imagick also.